app/renderer/components/preferences/payment/disabledContent.js

```diff
@@ -17,27 +20,63 @@ const uphold = require('../../../../extensions/brave/img/ledger/uphold_logo_smal
 const uphold2 = require('../../../../extensions/brave/img/ledger/uphold_logo_medium.png')
 
 class DisabledContent extends ImmutableComponent {
+  constructor (props) {
+    super(props)
+    this.text = this.getAlternativeText()
+  }
+
+  getAlternativeText () {
+    return <div>
+      <h3 data-l10n-id='paymentsWelcomeTitle' />
+      <p data-l10n-id='paymentsWelcomeText1' />
+      <p data-l10n-id='paymentsWelcomeText2' />
+    </div>
+  }
+
+  getText () {
+    if (this.props.ledgerData == null) {
+      return
+    }
+
+    const text = this.props.ledgerData.getIn(['promotion', 'panel', 'optInMarkup'])
+    const claimed = this.props.ledgerData.has('claimedTimestamp')
+
+    if (!text || !claimed) {
+      return
+    }
+
+    this.text = <div dangerouslySetInnerHTML={{ __html: text }} />
```

Comment:
we should avoid using dangerouslySetInnerHTML. could `optInMarkup` be modified to return text and templating data instead of raw HTML?